src/main/java/com/ericsson/ei/frontend/utils/BackEndInstanceFileUtils.java

```diff
@@ -92,4 +110,13 @@ private boolean fileContainsJsonArray() {
         }
     }
 
+    private void createEiHomeFolder() {
+        Boolean success = (new File(eiHome)).mkdirs();
+
+        if (!success) {
+           LOG.error("Failed to create eiffel intelligence home folder in {}. Please check access rights or change default folder in application.properties.".format(eiHome));
+           System.exit(-1);
```

Comment:
Maybe this should raise an Exception?

Reply:
Ye that might be better, i have no idéa which way to handle this the best, it should atleast not start the application if it's not possible to load/create user configurations.